src/hybrid/parse_hybrid.c

```diff
@@ -168,6 +168,14 @@ static int parseKNNClause(ArgsCursor *ac, VectorQuery *vq, ParsedVectorData *pvd
         QueryError_SetError(status, QUERY_ESYNTAX, "Invalid EF_RUNTIME value");
         return REDISMODULE_ERR;
       }
+      // Validate it's a positive integer
+      char *endptr;
+      errno = 0;
+      long long efValue = strtoll(value, &endptr, 10);
```

Comment:
Can replace the previous call to `AC_GetString` with `AC_GetLongLong` and avoid this snippet?